platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -89,6 +89,17 @@ Execution failed for task ':dependencies'.
 > Get more help at https://help.gradle.org.
 ```
 
+<a name="other-improvements"></a>
+
+### Other improvements
+
+#### Gradle encryption key via an environment variable
+
+You may now provide Gradle with the key used to encrypt cached configuration data via the `GRADLE_ENCRYPTION_KEY` environment variable. By default, Gradle creates and manages the key automatically, storing it into a keystore under the Gradle User Home, which may be inappropriate in some environments.
```

Comment:
```suggestion
You may now provide Gradle with the key used to encrypt cached configuration data via the `GRADLE_ENCRYPTION_KEY` environment variable.
By default, Gradle creates and manages the key automatically, storing it in a keystore under the Gradle User Home.
This may be inappropriate in some environments.
```